spec.html

```diff
@@ -38326,6 +38326,126 @@ <h1>`Promise.all` Resolve Element Functions</h1>
         </emu-clause>
       </emu-clause>
 
+      <emu-clause id="sec-promise.allsettled">
+        <h1>Promise.allSettled ( _iterable_ )</h1>
+        <p>The `allSettled` function returns a promise that is fulfilled with an array of promise state snapshots, but only after all the original promises have settled, i.e. become either fulfilled or rejected. It resolves all elements of the passed iterable to promises as it runs this algorithm.</p>
+        <emu-alg>
+          1. Let _C_ be the *this* value.
+          1. If Type(_C_) is not Object, throw a *TypeError* exception.
+          1. Let _promiseCapability_ be ? NewPromiseCapability(_C_).
+          1. Let _iteratorRecord_ be GetIterator(_iterable_).
+          1. IfAbruptRejectPromise(_iteratorRecord_, _promiseCapability_).
+          1. Let _result_ be PerformPromiseAllSettled(_iteratorRecord_, _C_, _promiseCapability_).
```

Comment:
```suggestion
          1. Let _result_ be ? PerformPromiseAllSettled(_iteratorRecord_, _C_, _promiseCapability_).
```

Reply:
This review feedback is wrong. It should not return if the return is an abrupt completion. 

The current text, without the `?`, is correct.